### ISSUE TITLE
NTR: Breast biopsy: fibrocystic disease  (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9143]
New term request Breast biopsy: fibrocystic disease  (elthat_100321184055):   UMLS: 0541809

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000896
- label: Fibrocystic breast lesion
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: PMID:31869073
- def: A type of benign cyst or cysts in breast tissue that are typically mobile within the glandular breast tissue, chest wall, and skin and are rubber-like in texture. The main histological features are an extracellular matrix of collagen, peri canalicular patterns of stromal cells with the presence of florid epithelial hyperplasia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9143
- synonyms:
